Content.Server/Hands/Systems/HandsSystem.cs

```diff
@@ -245,32 +247,32 @@ private void HandleSmartEquip(ICommonSession? session, string equipmentSlot)
             if (playerSession.AttachedEntity is not {Valid: true} plyEnt || !Exists(plyEnt))
                 return;
 
-            if (!TryComp<SharedHandsComponent>(plyEnt, out var hands))
+            if (!_actionBlockerSystem.CanInteract(plyEnt, null))
```

Comment:
Pretty sure this still runs that jank af event lol.

Reply:
Yeah it does. I've been meaning to have a go at fixing that, though it will probably make the mess of interaction events even worse